src/renderer/components/EmojiText.tsx

```diff
@@ -21,13 +21,12 @@ export const EmojiText: FC<IEmojiText> = ({ text }) => {
       ref.current.innerHTML = twemoji.parse(text, {
         folder: 'svg',
         ext: '.svg',
-        callback: (icon: string, options: TwemojiOptions, _variant: string) => {
```

Comment:
No longer need to pluck these out from the node_modules within the asar bundle 🚀